features/locality_load_balancing.md

```diff
@@ -0,0 +1,177 @@
+[//]: # (The syntax preceeding this line is a comment marker used to help guide the author in populating this document)
+[//]: # (to github. Unlike HTML comments commonly used throughout istio.io documentation, this comment will not be rendered)
+[//]: # (by github. Comments must be separated by carriage return preceding and concluding the text and be a single line.)
+
+[//]: # (This is a living document representing the maturity of a feature. Completion of this template enables Istio work groups)
+[//]: # (to collect information on potential new functionality. This template should be completed before users are exposed to)
+[//]: # (any new experimental feature. Please complete this template during development.)
+
+[//]: # (The feature implementation section must be completed before submission of the document.)
+
+# Feature: Locality Load Balancing [Beta]
+
+**Feature name:**
+
+Locality Load Balancing
+
+**Primary lead(s):**
+
+John Howard
+
+**Short description:**
+
+This feature helps direct/split traffic between services within a mesh(using Locality Fail-over and Locality-weighted load balancing) 
+to enable high availability and fast response time for workloads.
+  
+---
+
+## Experimental
+
+### Requirements:
+
+[//]: # (All information in this section is mandatory for promotion. Please modify the links in this)
+[//]: # (section.)
+
+- [ ] [User stories](insert_your_link_here) reviewed in a work group meeting.
+
+[//]: # (User stories are a way to communicate user value. User stories follow the style)
+[//]: # (as a [type of user], I want [an action] so that [a benefit/a value]. Istio currently has no user)
+[//]: # (story template. Maybe you can make one?)
+
+[//]: # (User stories must be presented in a work group meeting. They need no approval and are later integrated)
+[//]: # (into the RFCs, which do need approval for alpha. You may find value to negotiate within the work group where the)
+[//]: # (user stories are presented to help clarify the user stories.)
+
+- [ ] [RFC Authored] - [create an RFC using template](https://docs.google.com/document/d/1ewJoCcw5-04crH-M0xw4zFxz1cfwVCPnNyW4K3m4Yyc/template/preview).
+
+[//]: # (An RFC is mandatory to graduate to experimental. The RFC does not have to be reviewed in a work group)
+[//]: # (meeting to graduate to experimental.)
+
+- [ ] [Documentation](insert_your_link_here) for enabling and using the feature.
+
+[//]: # (The documentation instructions may exist on the developer wiki or the team drive. They may include instructions)
+[//]: # (for building running a `istioctl experimental command`, or using the preview profile,)
+[//]: # (or any other relevant information.)
+
+- [ ] [Feedback plan](insert_your_link_here).
+
+[//]: # (This may include user feedback meetings, discuss.istio.io conversations, GitHub issues, or mailing lists.)
+
+- [ ] Disabled by default.
+
+- [ ] No impact on performance when the feature is disabled.
+
+---
+
+## Alpha
+
+### Requirements: 
+
+**Design**
```

Comment:
Does this meet the requirements for alpha as well?

Reply:
I think everything here is a subset of beta, so should be met if beta is.

Reply:
That was my thinking as well. If it is in Beta, then it automatically covers Alpha as well. But I will go ahead and check the boxes that apply for alpha as well. That makes it clear if we have any gaps.

Reply:
This seems unchecked still